src/component/variantPage/BasicInfo.tsx

```diff
@@ -116,14 +119,25 @@ export default class BasicInfo extends React.Component<IBasicInfoProps> {
         const haveTranscriptTable = this.haveTranscriptTable(
             this.props.annotation
         );
+        const selectedTranscript =
+            this.props.annotation &&
+            _.find(
+                this.props.annotation.transcriptConsequenceSummaries,
+                consequenceSummary => {
+                    return (
+                        consequenceSummary.transcriptId ===
+                        this.props.selectedTranscript
+                    );
+                }
```

Comment:
minor but no need to have an explicit return here. you can shorten it like this:

```suggestion
                consequenceSummary => 
                    consequenceSummary.transcriptId === this.props.selectedTranscript
```